src/backends/sentry_backend_breakpad.cpp

```diff
@@ -37,38 +42,41 @@ sentry__breakpad_backend_send_envelope(
     const breakpad_transport_state_t *state
         = (const breakpad_transport_state_t *)_state;
 
-    sentry_path_t *dump_path = sentry__path_from_str(state->dump_path);
-    if (!dump_path) {
+    if (!state->dump_path) {
         sentry_envelope_free(envelope);
         return;
     }
     // when serializing the envelope to disk, and later sending it as a single
     // `x-sentry-envelope`, the minidump needs to be an attachment, with type
     // `event.minidump`
-    sentry_envelope_item_t *item
-        = sentry__envelope_add_from_path(envelope, dump_path, "attachment");
+    sentry_envelope_item_t *item = sentry__envelope_add_from_path(
+        envelope, state->dump_path, "attachment");
     if (!item) {
-        sentry__path_free(dump_path);
         sentry_envelope_free(envelope);
         return;
     }
     sentry__envelope_item_set_header(
         item, "attachment_type", sentry_value_new_string("event.minidump"));
-    sentry__envelope_item_set_header(item, "filename",
-        sentry_value_new_string(sentry__path_filename(dump_path)));
+
+#ifdef SENTRY_PLATFORM_WINDOWS
+    const char *file_name = sentry__path_filename_str(state->dump_path);
```

Comment:
ah, I completely forgot about the wstr on windows. I think we should rather do it like this here:

https://github.com/getsentry/sentry-native/blob/621ee38b0c6fa24a5bd035f66f8accda3ac1910f/src/sentry_core.c#L299-L305

instead of defining a new `_str` API.

Reply:
replaces